src/melee.cpp

```diff
@@ -1204,6 +1198,7 @@ void Character::roll_bash_damage( bool crit, damage_instance &di, bool average,
     bool unarmed = attack_vector != "WEAPON";
     int arpen = 0;
 
+    /** @EFFECTs documented below */
```

Comment:
Looking forward to a hopeful/potential future CI enforcement of comments for each use of a stat or skill. We'll need some way to say "this space intentionally left blank", and this is a placeholder for that.